changelog.json

```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.2.4-beta1": [
+      "[Fixed] Prevent application crash when spawn fails - #8614",
+      "[Fixed] Don't crash when attempting to use multibyte characters as name - #8534",
+      "[Fixed] Don't dismiss dialogs when dragging from the dialog to the backdrop - #8544",
+      "[Improved] Greater accuracy in branch protection checks - #7023"
```

Comment:
Considering this is the first time we've mentioned branch protection checks in the changelog, should change this line's category to `[New]`?

Reply:
took a swing at it in https://github.com/desktop/desktop/pull/8655/commits/0d8af10370647eade842a52f611778e012bc5ed0